redfish-core/lib/systems.hpp

```diff
@@ -3522,8 +3529,14 @@ inline void requestRoutesSystems(App& app)
                     if (ibmOem)
                     {
                         std::optional<bool> lampTest;
-                        if (!json_util::readJson(*ibmOem, asyncResp->res,
-                                                 "LampTest", lampTest))
+                        std::optional<bool> partitionSAI;
```

Comment:
Can we put these in the ifdef as well?

Reply:
> Can we put these in the ifdef as well?

Ack

Reply:
should the lamp test and SAI not be different and under their own #define ? In a case where we have SAI enabled and not lamp test, would we want to tread Json for lamp test ?